packages/upload-media/tsconfig.json

```diff
@@ -8,13 +8,13 @@
 	},
 	"include": [ "src/**/*" ],
 	"references": [
-	  { "path": "../api-fetch" },
```

Comment:
changes to this file should be reverted, probably a result of merge conflict or something

Reply:
The pre-commit Git hook is enforcing linting even after reverting changes.  Are there alternative solutions?

Reply:
I submitted a quick PR https://github.com/WordPress/gutenberg/pull/68083 to fix it separately.
Once that is merged, please rebase this PR.